mm2src/coins/eth.rs

```diff
@@ -3106,15 +3148,24 @@ fn signed_tx_from_web3_tx(transaction: Web3Transaction) -> Result<SignedEthTx, S
 pub struct GasStationData {
     // matic gas station average fees is named standard, using alias to support both format.
     #[serde(alias = "average", alias = "standard")]
-    average: f64,
+    average: MmNumber,
+    fast: MmNumber,
 }
 
 impl GasStationData {
-    fn average_gwei(&self, decimals: u8) -> U256 { U256::from(self.average as u64) * U256::exp10(decimals as usize) }
+    fn average_gwei(&self, decimals: u8) -> NumConversResult<U256> {
+        let average_fast_mean = (&self.average + &self.fast) / MmNumber::from(2);
```

Comment:
After discussing in private, Here it would be more interesting depending on the protocols used (ETH, Matic) to change the formula, Matic is rarely congested, the average should be sufficient in this case.

Reply:
Thanks for your review! I will add the `GasStationPricePolicy` enum that will customize this behavior and push the change today.